### DISCUSSION TITLE
Publish a 2.0.2 version - Android is broken

### DISCUSSION THREAD

**Discussion by gp3gp3gp3** [software-mansion/react-native-reanimated#1995] [Q&A]
This repo in its current published state is not functioning on Android.  The published version [2.0.1](https://github.com/software-mansion/react-native-reanimated/releases/tag/2.0.1) is broken because it does not include this commit to fix [full refreshing on Android](https://github.com/software-mansion/react-native-reanimated/pull/1839).  The latest version includes this, but also includes the Android [JSC](https://github.com/software-mansion/react-native-reanimated/pull/1842) and the extracted worklet logic.  I'm not sure which of those is causing the crash in [hermes](https://github.com/software-mansion/react-native-reanimated/issues/1923).

Normally I would just patch this, but they're in the android folder, so the files that need changes are not in the node modules.  If Software-mansion could make a patch release on 2.0.1 to _only_ include the full refreshing fix that would be ideal, since there is no way to debug android apps without crashes right now.

**Comment by mrousavy** [comment]
Why are you not using 2.1.0? Those commits landed in 2.1.0

**Reply by gp3gp3gp3** [reply]
Because Android debugger crashes when connected to hermes in 2.1.0

**Reply by gp3gp3gp3** [reply]
Sorry if I'm not making myself clear.  The refresh works in 2.1.0, but connecting to the hermes debugger in flipper causes a hard crash.  If there was a release that omitted the commits that cause the crash in hermes, which I am guessing has to do with the worklet or JSC issue, then Android builds could be connected via the flipper debugger and refreshed like normal.
